### DISCUSSION TITLE
Xpra appreciation thread

### DISCUSSION THREAD

**Discussion by wowczarek** [Xpra-org/xpra#3780] [General]
Me and screen or tmux never really got along. I i do light dev work and use lots of statistical and graphing packages and collect data sometimes for months - I mostly use Terminator and work from there, so I need X, and X lives on my Linux dev VM, while my work workstation is Windows. Xpra really was the holy grail - I can log in from anywhere and get my open windows back. This has been a tremendous quality of life improvement to my workflow. My xpra.conf starts Terminator which respawns in a loop should it die or be closed.

I decided to post this note here to thank all the developers involved, after I checked the uptime of my VM today. The uptime pretty much equals the Xpra session length, with the same Terminator instance still open that I use day in, day out. It's been running for four years now and counting. Would have been probably two years longer if it wasn't for a planned power outage exercise. One day I'll update it to the latest version...

So here's to you - you've been making my work easier for years now. Let me know how I can buy you a drink.

**Comment by totaam** [comment]
Thanks! This is very much appreciated.
https://github.com/sponsors/totaam

**Comment by totaam** [comment]
See also https://github.com/orgs/Xpra-org/discussions/3949

**Comment by KondorIstvan** [comment]
+1

I started using WinSwitch as a student when studying abroad to reach my home computer.
This is one of the most important OSS projects for me personally, which is not yet backed by some large company.

@totaam Your work, quick support, timely releases and so on are highly appreciated! I've been a vocal OSS advocate for a long time but never used 'buy me coffee' or similar services. I'll look into getting a six-pack for you, though!

**Comment by EgosOwn** [comment]
@totaam Thank you for so many releases recently, I can tell you have been working very hard.  

xpra has enabled me not only to have "persistent remote applications" but also get QubesOS-like seamless windowing from my libvirt vms and containers - truly under used and under appreciated software.
